parts/linux/cloud-init/artifacts/ipv6_nftables.sh

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+
+set -uo pipefail
+set -x
+
+NFTABLES_RULESET_FILE=/etc/systemd/system/ipv6_nftables
+
+# query IMDS to check if node has IPv6
+# example interface
+# [
+#   {
+#     "ipv4": {
+#       "ipAddress": [
+#         {
+#           "privateIpAddress": "10.224.0.4",
+#           "publicIpAddress": ""
+#         }
+#       ],
+#       "subnet": [
+#         {
+#           "address": "10.224.0.0",
+#           "prefix": "16"
+#         }
+#       ]
+#     },
+#     "ipv6": {
+#       "ipAddress": [
+#         {
+#           "privateIpAddress": "fd85:534e:4cd6:ab02::5"
+#         }
+#       ]
+#     },
+#     "macAddress": "000D3A98DA20"
+#   }
+# ]
+
+# every 60 min, query IMDS and update nftables rules
```

Comment:
not a fan of this right now. need to understand why our rules aren't working after 2 hours.

Reply:
fixed